tox.ini

```diff
@@ -1,15 +1,18 @@
 [tox]
 envlist =
-    django111-{py27,py36},
-    django110-{py27,py36},
-    django19-{py27,py36},
+    django111-py27
+    {django111,django20,django21}-{py35,py36,py37}
 
 [testenv]
 commands =
-    coverage run manage.py test export.tests --settings=test_settings
-    coverage report -m
+    coverage run manage.py test
 deps =
     coverage
     django111: django>=1.11,<2.0
-    django110: django>=1.10,<1.11
-    django19: django>=1.9,<1.10
+    django20: django>=2.0,<2.1
```

Comment:
Seeing that you have these here, all the requirements files for the tests can be removed